semeio/workflows/localisation/local_script_lib.py

```diff
@@ -1,4 +1,5 @@
 # pxylint: disable=attribute-defined-outside-init
+# pylint: disable=C0302
```

Comment:
can you spell this one out so we don´t have to look it up?

Reply:
Will try.